networking/configuring-a-custom-pki.adoc

```diff
@@ -40,6 +40,12 @@ The `trustedCA` field is a reference to a `ConfigMap` containing the custom
 certificate and key pair used by the cluster component.
 ====
 
+== Alternatives for additional trust
+
+Cluster components may provide alternative mechanisms for configuring additional trust, for situations when a proxy is not needed:
+
+* xref:../builds/setting-up-trusted-ca.adoc[Builds]
```

Comment:
It's better for the link to use the doc title: s/Builds/Setting up additional trusted certificate authorities for builds